profiles/gambhirsharma.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/69895353/184379082-e5704e10-e321-4742-a779-137751a95794.png",
+  "issueId": 1298,
+  "name": "Gambhir Sharma",
+  "username": "gambhirsharma"
+}
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable